tests/output/verify.py

```diff
@@ -0,0 +1,30 @@
+#!/bin/env python
+
+# Copyright 2019 Arm Limited.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser(description='Test generator.')
+parser.add_argument('--out')
+parser.add_argument('--expected')
+
+args = parser.parse_args()
+
+if os.path.basename(args.out) != args.expected:
+    print(os.path.basename(args.out))
+    msg = "Output from generation: {} but expected: {}".format(args.out, args.expected)
+    raise Exception(msg)
```

Comment:
Pref: Raising an exception here will mean we get an unnecessary Python backtrace when this test fails.

Instead, can you just do `sys.exit(1)` (and print the message obviously :) )?

Reply:
Github's also complaining about the missing `\n` here too :(

Reply:
Well I think it's cleaner use dedicated raise exception form than print based traceback?

Reply:
Thanks!

The issue is that we don't need a backtrace. It's the same as e.g. if I do this:
```bash
# cat non_existent_file
cat: non_existent_file: No such file or directory
```
I get a single-line error message - I don't get a backtrace from `cat`, because the problem is with the user/whatever is running `cat`, not `cat` itself.

A backtrace/Python exception is only useful when there is a bug in the Python itself - but this "output from generation..." message is demonstrating that there's an issue with Bob, *not* with `verify.py` :)